prow/tide/README.md

```diff
@@ -0,0 +1,131 @@
+# Tide
+
+Tide is a [Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md)
+component for managing a pool of PRs that match a given set of criteria.
+Automatically retesting PRs in the pool and merging them if they pass tests
```

Comment:
s/Automatically retesting PRs in the pool and merging/It will automatically retest and merge PRs in the pool/

Reply:
I am fan of the first form :) It sounds more poetic. Though, the latter is fine too :).